wordpress-mobile/gutenberg-mobile
# Temporary gutenberg update to test media files type

Fixes an error when the media files delegate methods are called on iOS.

See https://github.com/wordpress-mobile/WordPress-iOS/pull/15811
See https://github.com/WordPress/gutenberg/pull/28766

<a href="https://user-images.githubusercontent.com/3250/107052017-66a26a80-678a-11eb-9a90-80b6468f8d14.png"><img src="https://user-images.githubusercontent.com/3250/107052017-66a26a80-678a-11eb-9a90-80b6468f8d14.png" width="200"></a>

To test:
* Open a post containing a stories block
* Attempt to edit the block
* With this change, no error should be shown
* Without this change, an error screen is shown (see above)

PR submission checklist:

- [x] ~I have considered adding unit tests where possible.~ Didn't see a place to test this
- [x] !I have considered if this change warrants user-facing release notes.~ Unreleased on iOS.
